pkg/pillar/cmd/volumemgr/initialvolumestatus.go

```diff
@@ -2,22 +2,20 @@
 // SPDX-License-Identifier: Apache-2.0
 
 // Handle publishing the existing-at-boot VolumeStatus
-// Published under "unknown" objType with refcount=0. Moved to
-// other objType when there is a reference.
+// Published under "unknown" objType
```

Comment:
Is the old comment about refcount=0 no longer correct?

Reply:
We need refcount. Corrected.